executor/adapter.go

```diff
@@ -1247,7 +1247,11 @@ func FormatSQL(sql string) stringutil.StringerFunc {
 			return QueryReplacer.Replace(sql) // no limit
 		}
 		if int32(length) > maxQueryLen {
-			sql = fmt.Sprintf("%.*q(len:%d)", maxQueryLen, sql, length)
+			var result strings.Builder
+			result.Grow(int(maxQueryLen))
+			result.WriteString(fmt.Sprintf("%q", sql)[:maxQueryLen])
+			result.WriteString(fmt.Sprintf("(len:%d)", length))
```

Comment:
The fmt.Sprintf seems unnecessary, use `sql[:maxQueryLen]` directly?
And `fmt.Fprintf(result, "(len:%d)", length)`

Reply:
Yes, I think too. Furthermore, I have asked the original author and they cannot remember the reason for this formatting. Therefore, I have decided to remove this format